calibrators/mapping_based_calibrator/README.md

```diff
@@ -0,0 +1,238 @@
+| `detected_objects`               | `autoware_perception_msgs::msg::DetectedObjects`  | Messages containing detected objects, used in the filtering procedure.                                                              |
+| `predicted_objects`              | `autoware_perception_msgs::msg::PredictedObjects` | Messages that contain predicted object paths and positions, used in the filtering procedure.                                        |
+
+### Output
+
+| Name                              | Type                                   | Description                                                                                                    |
+| --------------------------------- | -------------------------------------- | -------------------------------------------------------------------------------------------------------------- |
+| `output_map`                      | `sensor_msgs::msg::PointCloud2`        | Output map constructed from the `mapping_pointcloud`.                                                          |
+| `frame_path`                      | `nav_msgs::msg::Path`                  | The actual path of the `mapping_pointcloud`.                                                                   |
+| `frame_predicted_path`            | `nav_msgs::msg::Path`                  | The predicted path of the`mapping_pointcloud`.                                                                 |
+| `keyframe_path`                   | `nav_msgs::msg::Path`                  | The keyframe path of the `mapping_pointcloud`.                                                                 |
+| `keyframe_markers`                | `visualization_msgs::msg::MarkerArray` | Markers for keyframes.                                                                                         |
+| `initial_source_aligned_map`      | `sensor_msgs::msg::PointCloud2`        | Initial map from calibration lidars.                                                                           |
+| `calibrated_source_aligned_map`   | `sensor_msgs::msg::PointCloud2`        | Calibrated map from calibration lidars.                                                                        |
+| `target_map`                      | `sensor_msgs::msg::PointCloud2`        | Target map from `mapping lidar`, used for comparing with the `calibrated_source_aligned_map` and `target_map`. |
+| `target_markers`                  | `visualization_msgs::msg::MarkerArray` | Markers for camera calibrator (currently not used)                                                             |
+| `base_lidar_augmented_pointcloud` | `sensor_msgs::msg::PointCloud2`        | Ground pointcloud from augmented pointcloud.                                                                   |
```

Comment:
grammar (applies to the next line as well)

Reply:
fixed in https://github.com/tier4/CalibrationTools/pull/164/commits/07225d64a26ff4ac5e7fb2945ec97463490aa6bd